src/include/common/string_utils.h

```diff
@@ -69,6 +69,8 @@ class StringUtils {
         str = std::regex_replace(str, whiteSpacePattern, "");
     }
 
+    static const char* removeCStringWhiteSpaces(const char* input, uint64_t& len);
```

Comment:
If `len` is passed by reference, do you think `void removeCStringWhiteSpaces(const char *& input, uint64_t& len)` makes more sense?